src/ensembl/production/core/perl_utils.py

```diff
@@ -59,12 +60,14 @@ def list_to_perl_string(input_list):
 
 def escape_perl_string(v):
     """Escape characters with special meaning in perl"""
-    return str(v).replace("$", "\\$").replace("\"", "\\\"").replace("@", "\\@")
+    s = str(v).replace("$", "\\$").replace("\"", "\\\"").replace("@", "\\@")
+    return re.sub('[\n\t\r]+', '', s)
```

Comment:
Looks like a typo, as I don't see why completely removing these blanks ending up with conversions like `"name\tdate"` -> `"namedate"`

```suggestion
    return re.sub(r'[\n\t\r]+', ' ', s)
```